_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -7516,17 +7516,17 @@ void MfxVideoParam::SyncVideoToCalculableParam()
 
     if (IsMvcProfile(mfx.CodecProfile))
     {
-        mfxExtMVCSeqDesc * extMvc = GetExtBuffer(*this);
-        if (extMvc->NumView)
+        mfxExtMVCSeqDesc & extMvc = GetExtBufferRef(*this);
```

Comment:
It's not a good idea to use this method inside class MfxVideoParam - there is no 100% guarantee that needed buffer is existed, so there is a way to undefined behavior.
As result, if pointer extMvc can be nullptr, it's better to check it directly, for example in the "if" condition at line 7520

Reply:
Here starts the list of buffers which can be safely extracted with GetExtBufferRef: https://github.com/Intel-Media-SDK/MediaSDK/blob/dd25daaa1ff50d8ee41f4c14b8599db3e11d2331/_studio/mfx_lib/shared/include/mfx_h264_enc_common_hw.h#L547

mfxExtMVCSeqDesc is not in this list so you can't use GetExtBufferRef for it